packages/components/organisms/f-checkout/src/event-names.js

```diff
@@ -4,12 +4,14 @@ const CheckoutGetSuccess = 'checkout-get-success';
 const CheckoutGetFailure = 'checkout-get-failure';
 const CheckoutAvailableFulfilmentGetSuccess = 'checkout-available-fulfilment-get-success';
 const CheckoutAvailableFulfilmentGetFailure = 'checkout-available-fulfilment-get-failure';
+const CheckoutVisitLoginPage = 'checkout-visit-login-page';
```

Comment:
This need to be added to the README.md events description along with ccs class you've added:

https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-checkout/README.md#events
https://github.com/justeat/fozzie-components/blob/master/packages/components/organisms/f-checkout/README.md#css-classes